### DISCUSSION TITLE
Codespaces got error 403 when open and push to a fork

### DISCUSSION THREAD

**Discussion by haifeng-jin** [community/community#36357] [Codespaces]
I am a maintainer, a contributor opened a pull request, which `allow edits from maintainers`.
When following [this guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork) to modify the contributor's branch, everything works fine.

However, if I do not follow from step 3 to step 9 (which is just clone the contributor's forked repo locally and checkout the branch for the pull request) but go to the forked repo and switch to the branch on the web UI and open in Codespaces, I will get error 403 if I try to push to the remote (which is the contributor's branch).

**Comment by cwndrws** [comment]
Thanks for reporting the issue. This seems to be a recent regression and we're tracking the issue internally.
